smallfile_wrapper/trigger_smallfile.py

```diff
@@ -5,6 +5,7 @@
 import shutil
 import socket
 from vfs_stat import get_vfs_stat_dict
+import redis
```

Comment:
Can you please add this library to the requirements as well ? https://github.com/cloud-bulldozer/snafu/blob/master/requirements.txt